Recast/Cells/PodcastGridTableViewCell.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  SeriesGridTableViewCell.swift
+//  Recast
+//
+//  Created by Jaewon Sim on 9/22/18.
+//  Copyright © 2018 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+class PodcastGridTableViewCell: UITableViewCell {
+    private var collectionView: UICollectionView!
```

Comment:
marker! also do they need to be private? @drewsdunne when should we use `private var`s?